Source/Cogworks.Essentials/Extensions/StringExtensions.cs

```diff
@@ -15,7 +15,9 @@ public static bool HasValue(this string input)
             => !string.IsNullOrWhiteSpace(input);
 
         public static Uri ToUri(this string urlString)
-            => new Uri(urlString);
+            => urlString.HasValue()
```

Comment:
This was done by purpose, uri will throw exception if string null or empty.

Reply:
uh ??

Reply:
Sorry read it wrong on beginning. Looks good. If empty or null we are returning null 👍 all good